tasks/src/main/java/org/example/aProblemAboutEquation/AProblemAboutEquation3.java

```diff
@@ -0,0 +1,79 @@
+package org.example.aProblemAboutEquation;
+
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.io.InputStreamReader;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.StringTokenizer;
+
+
+/**
+ * https://codeforces.com/problemset/problem/174/A
```

Comment:
zastap to jednolinijkowym komentarzem, 3 linie to zbedna rzecz. To samo te puste liine na gorze. Wystarczy jedna